docs/en/guides/ros-quickstart.md

```diff
@@ -0,0 +1,377 @@
+---
+comments: true
+description: todo
+keywords: Ultralytics, YOLO, object detection, deep learning, machine learning, guide, ROS, Robot Operating System, robotics, ROS Noetic, Python, Ubuntu, simulation, visualization, communication, middleware, hardware abstraction, tools, utilities, ecosystem, Noetic Ninjemys,compatibility, performance, stability
```

Comment:
`Noetic Ninjemys,compatibility` add space after `,`

Reply:
not sure if compatibility, performance, stability are completely relevant here. Maybe add "autonomous vehicle", "AMV", "autonomous motorized vehicle", etc.